.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+    "singleQuote": true,
+    "useTabs": true,
+    "tabWidth": 1,
```

Comment:
I think this `tabWidth` should be `2` instead of 1. Could you guide why this is 1?

Reply:
> @shubhangi013
> Please read the contributing guidelines.
> Currently, you are on master. You should create changes in a separate branch and then make a PR. It's fine for this but be careful next time.

I'm sorry. will keep that in mind for the next time. Thankyou

Reply:
> I think this `tabWidth` should be `2` instead of 1. Could you guide why this is 1?

yeah, should've been 2. Fixing it.